public/service_portal/v0.1.0/openapi.json

```diff
@@ -428,6 +428,49 @@
                 }
             }
         },
+        "/portfolio_items/{portfolio_item_id}/icon": {
+            "get": {
+                "tags": [
+                    "users",
+                    "admins"
+                ],
+                "summary": "Fetches the specified portfolio item's icon information, reaching out to topology if necessary",
```

Comment:
Small nit, but I don't think we need to add the Topology bit as I'm not sure there is a guarantee that the reader would know what that means or not.

Reply:
Done. Makes sense, that way it's transparent to the end-user where the icon information is coming from.